nxos_bgp.tf

```diff
@@ -187,3 +187,38 @@ resource "nxos_bgp_peer_address_family" "bgpPeerAf" {
   send_community_extended = each.value.send_community_extended
   send_community_standard = each.value.send_community_standard
 }
+
+locals {
+  routing_bgp_vrfs_neighbors_address_families_route_control = flatten([
+    for device in local.devices : [
+      for vrf in try(local.device_config[device.name].routing.bgp.vrfs, []) : [
+        for nei in try(vrf.neighbors, []) : [
+          for af in try(nei.address_families, []) : (
+            try(af.route_map_name, null) != null ? [
+              {
+                key            = format("%s/%s/%s/%s", device.name, vrf.vrf, nei.ip, local.address_family_names_map[af.address_family])
+                device         = device.name
+                vrf            = vrf.vrf
+                neighbor_key   = format("%s/%s/%s", device.name, vrf.vrf, nei.ip)
+                address_family = local.address_family_names_map[af.address_family]
+                route_map_name = af.route_map_name
```

Comment:
Assuming we can have both, an "in" and "out" route map, should we maybe have two attributes, `route_map_in` and `route_map_out`? (We avoid appending a `_name` suffix as we typically refer to other objects by its name)

Reply:
My latest commit should address this.